lib/repository.py

```diff
@@ -101,7 +101,7 @@ def run(self):
             scripts = []
             print("Downloading scripts...")
             for line in version_file:
-                scripts.append(line.strip('\n').split(','))
+                scripts.append(line.decode("utf-8").strip('\n').split(','))
```

Comment:
I think `line.decode().strip('\n').split(',')) ` would work. Since Latin-1 is the default.